homeassistant/components/bosch_shc/sensor.py

```diff
@@ -117,7 +117,7 @@ async def async_setup_entry(
         )
 
     for sensor in (
-        session.device_helper.smart_plugs + session.device_helper.light_switches
+        session.device_helper.smart_plugs + session.device_helper.light_switches_bsm
```

Comment:
Is this change due to changes in boschshcpy?

Reply:
Yes. The interface to access these devices in boschshcpy has changed